### ISSUE TITLE
Accessibility: Operable - Keyboard Accessible

### ISSUE THREAD

**Issue by AntonLV** [unacms/UNA#5046]
The website needs to be fully accessible via keyboard. Websites that can be operated using only the keyboard can also be used with alternative keyboards and voice input technologies.
All functionalities of the website are operable using the keyboard.

**Rationale**:

- After opening the submenu in the main navigation, keyboard focus cannot be moved to the expanded submenu items. As a result, users are unable to reach or operate these menu points using the Tab key.

- If an `<a>` element does not include an `href` attribute, it is not a proper link and therefore inaccessible to keyboard users. One such element appears on the homepage:

```<a name="more"></a>```

- An `<a>` without `href` cannot be reached via the Tab key, as it is no longer technically considered a link.

- The use of the `name` attribute for anchors is deprecated. Historically, `name="..."` was used for in-page anchors, but today the standard approach is to use `id="..."`. 

**Recommendation**:

- Make the main navigation operable via keyboard.
